src/main/scala/Chisel/util/Decoupled.scala

```diff
@@ -22,21 +22,29 @@ object Decoupled {
   def apply[T <: Data](gen: T): DecoupledIO[T] = new DecoupledIO(gen)
 }
 
-/** An I/O bundle for enqueuing data with valid/ready handshaking */
+/** An I/O bundle for enqueuing data with valid/ready handshaking
```

Comment:
Consider adding punctuation so people don't read this as one giant sentence, but rather as two.
Some style guide actually recommend that all comments are sentences, complete with periods...